scripts/missions/windurst/1_1_The_Horutoto_Ruins_Experiment.lua

```diff
@@ -0,0 +1,393 @@
+            ['Puo_Rhen'] = mission:event(88),
+            ['Ten_of_Clubs'] = mission:event(87),
+        },
+
+        [xi.zone.WINDURST_WALLS] =
+        {
+            ['Keo-Koruo'] = mission:event(100),
+            ['Pakke-Pokke'] = mission:event(101),
+            ['Zokima-Rokima'] = mission:event(99),
+        },
+
+        [xi.zone.WINDURST_WATERS] =
+        {
+            ['Dagoza-Beruza'] = mission:event(121),
+            ['Mokyokyo'] = mission:event(123),
+            ['Panna-Donna'] = mission:event(122),
+            ['Ten_of_Hearts'] = mission:event(124),
+        },
+
+        [xi.zone.WINDURST_WOODS] =
+        {
+            ['Miiri-Wohri'] = mission:event(124),
+            ['Rakoh_Buuma'] = mission:event(127),
+            ['Sola_Jaab'] = mission:event(125),
+            ['Tih_Pikeh'] = mission:event(126),
+        },
+    },
+
+    -- Go to the Orastery in Port Windurst (Home Point #1) and talk to Hakkuru-Rinkuru (E-7) for a cutscene.
+    {
+        check = function(player, currentMission, missionStatus)
+            return currentMission == mission.missionId and missionStatus == 1
+        end,
+
+        [xi.zone.PORT_WINDURST] =
+        {
+            ['Hakkuru-Rinkuru'] = mission:progressEvent(90),
+
+            onEventFinish =
+            {
+                [90] = function(player, csid, option, npc)
+                    player:setMissionStatus(mission.areaId, 2)
+                end,
+            },
+        },
+    },
+
+    -- Head to Inner Horutoto Ruins at (J-7) and examine Gate: Magic Gizmo
+    {
+        check = function(player, currentMission, missionStatus)
+            return currentMission == mission.missionId and missionStatus == 2
+        end,
+
+        [xi.zone.EAST_SARUTABARUTA] =
+        {
+            ['Sama_Gohjima'] = mission:event(53),
+        },
+
+        [xi.zone.PORT_WINDURST] =
+        {
+            ['Hakkuru-Rinkuru'] = mission:event(91),
+        },
+
+        [xi.zone.INNER_HORUTOTO_RUINS] =
+        {
+             -- Gate: Magical Gizmo
+            ['_5c5'] =
+            {
+                onTrigger = function(player, npc)
+                    return mission:progressEvent(42)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [42] = function(player, csid, option, npc)
+                    player:setMissionStatus(mission.areaId, 3)
+                    mission:setVar(player, 'RandomGizmo', math.random(1, 6))
+                end,
+            },
+        },
+    },
+
+    -- Find the broken Gizmo
+    {
+        check = function(player, currentMission, missionStatus)
+            return currentMission == mission.missionId and missionStatus == 3
+        end,
+
+        [xi.zone.INNER_HORUTOTO_RUINS] =
+        {
+            ['_5cp'] = -- Magical Gizmo #1
+            {
+                onTrigger = function(player, npc)
+                    return examineGizmo(mission, player, 1, 48, 49)
```

Comment:
Just following up here; examineGizmo shouldn't need to have mission parameter in its list, or passed.